e2e/Dockerfile

```diff
@@ -3,20 +3,27 @@ FROM mcr.microsoft.com/playwright:v1.43.1-jammy as base
 ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
 ENV NODE_ENV=test
+RUN apt-get update && apt-get install -y netcat
 WORKDIR /srv/apps/atj-platform
-COPY . .
+COPY ./pnpm-lock.yaml ./pnpm-lock.yaml
+COPY ./package.json ./package.json
 RUN corepack enable
-RUN pnpm install --filter=@atj/spotlight --frozen-lockfile
 
 FROM base as test
 ENV E2E_ENDPOINT=http://localhost:9090
+ENV CI=true
+COPY . .
 RUN npm install -g serve
 EXPOSE 9090
-RUN pnpm build --filter=@atj/spotlight
+EXPOSE 9191
+RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --frozen-lockfile
+RUN pnpm build --filter=@atj/spotlight --filter=@atj/design
 WORKDIR ./e2e
-RUN serve ../apps/spotlight/dist -p 9090 & sleep 5; pnpm playwright test;
+RUN serve ../apps/spotlight/dist -p 9090 -L & while ! nc -z localhost 9090; do sleep 1; done; pnpm playwright test;
+RUN serve ../packages/design/storybook-static -p 9191 -L & while ! nc -z localhost 9191; do sleep 1; done; pnpm --filter=end-to-end-tests test:storybook --url http://localhost:9191 --config-dir ../packages/design/.storybook/ --browsers firefox chromium
```

Comment:
I'm good to give this a try... It solves the complaint about not being able to run locally, and gets the job done pretty concisely.

After using awhile, I might prefer to (for local development) `pnpm install playwright` and run tests natively on my Mac. With the container, running the tests in the multi-stage build portion is interesting and a nice way to keep it repeatable... but running the server during the build means we'd lose the ability to mount a volume on the container at runtime, and do dev server hot-reloading kinds of behavior.